### ISSUE TITLE
DoorBell- Information 

### ISSUE THREAD

**Issue by aalzooke** [zjzsliyang/Transfiguration#2]
Would you please explain how to run the entire application also please tell me what is the hardware requirements, it will be helpful if you can provide the parts name. 
Thank you

**Comment by zjzsliyang** [comment]
We use The Raspberry Pi to run our server on the back of the door, you can also use other single-chip microcomputers. Thus, you can choose a camera for face recognition or Matrix Keypad for password input.

**Comment by aalzooke** [comment]
Thank you for your response, I'm having a hard time to communicate between the app and the server that runs on Amazon cloud9. the server port number is 3000 and URL app is http://60.205.206.174::3000. 
Do I need to change the URL so both app and server can communicate?

**Comment by zjzsliyang** [comment]
Yep, the URL is our server which runs in the Alibaba Cloud. You should change it to your own Amazon cloud IP Address.

**Comment by aalzooke** [comment]
Which product did you choose for server in Alibaba cloud? also which product do you recommend to use for the server in Amazon cloud?

**Comment by aalzooke** [comment]
Also are you willing to do extra work for this project and i will pay you?

**Comment by aalzooke** [comment]
Please let me know once you have time
